lib/tasks/panopticon.rake

```diff
@@ -103,7 +105,16 @@ namespace :panopticon do
           registerer.register(artefact)
         rescue GdsApi::HTTPErrorResponse => e
           logger.error "Failed to register /#{edition.slug} with #{e.code}: #{e.error_details}"
+          unregistered_documents << "#{edition.slug}, error code: #{e.code}, error details: #{e.error_details}"
+        rescue StandardError => e
+          logger.error "Failed to register /#{edition.slug}, error: #{e}"
+          unregistered_documents << "#{edition.slug}, error: #{e}"
         end
+
+        puts
+        puts "*******************************"
+        puts "Slugs of unregistered documents along with the errors:"
+        puts unregistered_documents
```

Comment:
I think this set of lines should be outside of the loop, otherwise it will be printed for every single document?